tests/openthread/Makefile

```diff
@@ -8,11 +8,6 @@ BOARD_WHITELIST := samr21-xpro iotlab-m3 fox iotlab-a8-m3
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
-# Comment this out to disable code in RIOT that does safety checking
-# which is not needed in a production environment but helps in the
-# development process:
-CFLAGS += -DDEVELHELP -Wall
```

Comment:
`-Wall` got removed at the same time here

Reply:
IMHO wasn't appropriate place to activate that here anyway, use `WERROR` (see e.g. #7919) instead